Example/Example/Model/Model.swift

```diff
@@ -1,7 +1,7 @@
 import Faro
 
 /// Example model
-class Model: Parseable {
+class Model: Deserializable {
```

Comment:
DOES NOT COMPUTE...

![](http://gif.co/w5Gt.gif)

Ik snap de `Deserializable` move totaal niet eigenlijk... Het was toch goed?